### ISSUE TITLE
NCAA14 - Issues needing fixed(current 8-31-19)

### ISSUE THREAD

**Issue by huskereurocat** [RPCS3/rpcs3#6400]
There are currently three issues with NCAA14 that, IMO, if fixed would make the game completely playable. From time to time we have game play that gets slow, but these are the major issues below. 
1. There are no face masks on the players.
2. Sound is choppy during plays. I thought it was the crowds getting loud, but I turned off the crowd noise and the issue is still present in the game. 
3. Cold weather breath is visible in blocky form. This is to the point that it blocks to view of the line of scrimmage on some plays and makes seeing where to run/pass the ball nearly impossible. 
Issues 1 and 3 were fixed at one time. Digitaldude555 posted on Wednesday 8-21-19 at 3.03pm in media pics of these issues being fixed and the current build 8568 doesn't have them in it. 
Here are pics of what 1 and 3 look like. 
![Desktop Screenshot 2019 08 24 - 10 30 18 35](https://user-images.githubusercontent.com/52609668/63639678-ff5b2e00-c65b-11e9-8d69-8bf68542d285.png)
![Desktop Screenshot 2019 08 24 - 10 31 34 97](https://user-images.githubusercontent.com/52609668/63639679-ff5b2e00-c65b-11e9-853f-b816ee1e2db4.png)
![Desktop Screenshot 2019 08 24 - 10 45 33 17](https://user-images.githubusercontent.com/52609668/63639707-637df200-c65c-11e9-9edd-1a13e02877ef.png)

**Comment by Asinin3** [comment]
Umm wasn't this all fixed by https://github.com/RPCS3/rpcs3/pull/6388 ? DigitalDude555 Reported it see: https://cdn.discordapp.com/attachments/272875751773306881/613825587525451776/Screenshot_502.png  https://cdn.discordapp.com/attachments/272875751773306881/613827238332203019/Screenshot_507.png etc

**Comment by huskereurocat** [comment]
Sorry, for all I knew 1 and 3 were skipped. Didn't realize that the open PR was not merged yet. This, most likely, will be with the next master build released? 
What log do you need to look at and I will post that in #help if that is where it belongs?

**Comment by CrabLugo** [comment]
Just go to help channel and ask for help with sound ingame

**Comment by huskereurocat** [comment]
I just received a comment from DigitalDude555 that said the facemasks were not fixed and he said they randomly were fixed in his game. I play the game almost daily, if nothing else, to find out if a new build fixed the issue. 
#1 Not yet fixed. No face masks.
#2 Sound is still choppy. Since I was told that there could be starved audio threads, could there be a setting that is not correctly set that could be changed? 
#3 Has been completely fixed. Yea!!!
Thanks for your help!!

**Comment by huskereurocat** [comment]
Issue #1 has been resolved by PR #6485. It looks freakin' awesome!!!!!!!!!!
That just leaves the sound issue, Please let me know if that should be entered in a different issue report. If so this report could be closed.

**Comment by kd-11** [comment]
Audio issues are often just caused by wrong settings. Usually "SPU loop detection" or "Lower SPU thread priority" are the cause.

**Comment by huskereurocat** [comment]
![Desktop Screenshot 2019 09 06 - 16 39 37 23](https://user-images.githubusercontent.com/52609668/64462150-1b7fb600-d0c5-11e9-84ef-89f63eeecbf5.png)
I was encouraged to turn both off and are already off. Still having the sound issue.

**Comment by kd-11** [comment]
That leaves two possibilities: Your CPU doesn't have enough cores or the SPU thread options are causing the issue. Try SPU threads on auto and even try ASMJIT decoder just in case. But in the end if you don't have enough threads, it will crackle under windows.

**Comment by huskereurocat** [comment]
I have a r7 1700 so the # of threads shouldn't be an issue. Let me try SPU threads on auto and ASMJIT and see if that does the trick.

**Comment by huskereurocat** [comment]
Here are my new settings per your message. 
![Desktop Screenshot 2019 09 06 - 17 21 11 74](https://user-images.githubusercontent.com/52609668/64463764-d1013800-d0ca-11e9-95c4-183a878083dd.png)


This is what the sound is like. Didn't change from the old settings.
[https://www.mediafire.com/file/domt3161vkfdn4n/rpcs3.exe_2019.09.06_-_17.05.59.01.mp4/file](url)

**Comment by dio-gh** [comment]
Needs testing by someone with a strong CPU (think i7 7700(K) / 8700K / 9900K). I'm fairly sure that's a performance limitation and not an actual emulation issue.

**Comment by huskereurocat** [comment]
And if it's a performance limitation, we're just stuck with the popping and stuttering sound? Sorry just don't quite understand?

**Comment by dio-gh** [comment]
Basically, yes. You may try doing the following before giving up however:

- try fine tuning the `Audio Buffer Duration` setting in the `Audio` tab. It's `100 ms` by default, you may try upping it to something like `130 ms` or `150 ms`. It basically makes the emulator buffer up a certain amount of audio before starting to play it; beyond `150 ms` it's going to be noticeably behind the visuals though, so that's not recommended.
- try disabling `Audio Buffering` (it's at the same place). This may sound counter intuitive, but some games don't quite work with audio buffering, so it's always worth trying.
- try altering the number of `Preferred SPU Threads`, the `SPU Block Size` and the `Maximum Number of SPURS Threads` options. The first two are in the `CPU` tab, the last one is in the `Advanced` tab. For the SPURS option, you want to go down one-by-one starting from 5, and see if anything happens. The game may or may not appreciate changing this option however (it might lock up / crash / etc.). And as for block size, I usually roll with `Safe`, but gotta try them all :p
- and if all this fails, you may try playing around with xfloat accuracy. Right click on the game and select `Open Custom Config Folder`. This will lead you to the game-specific config file RPCS3 created (should be named like `config_BLUS31159.yml`). Open it up in a text editor, and look for the `Accurate xfloat` and `Approximate xfloat` lines. Set both to false first and reboot the game (the emulator may remain open while doing this). Disabling `Approximate xfloat` might give you a performance uplift, and thus, curb the sound stutter. If this didn't pan out, enable `Accurate xfloat` (you can do this one on the `CPU` tab too). The game might need accurate xfloats to drive sound properly for all we know. :man_shrugging:

If you tried all this in every sensible combination and the sound still remained :hankey:, then yes, you're stuck with it for now. You'll need to wait for SPU performance improvements or buy better hardware to be able to enjoy this game properly.

**Comment by huskereurocat** [comment]
Well, I went through and systematically tested every combination that I could think of and the final analysis is that I guess I'll have to play the waiting game. Messing with the approx and accurate xfloat configurations is the only difference that I noticed and it made it worse. Hopefully SPU improvements will be made soon. I can hardly wait for that to happen. It will make a huge impact in the quality of my video journal on YT. I do have to say that the addition of the face masks is nothing but a game changer in the graphics arena. Hooray!!!!!!
